src/Morphir/TypeScript/AST.elm

```diff
@@ -15,11 +15,17 @@ that we use in the backend.
 
 -}
 
+import Morphir.IR.FQName exposing (FQName)
+import Morphir.IR.Name exposing (Name)
+import Morphir.IR.Path exposing (Path)
+
```

Comment:
I like this approach of re-importing types from Morphir.IR into our own namespace, so we can replace them later if desired.